src/leo_mq_api.erl

```diff
@@ -290,9 +290,7 @@ start_child_2(RefSup, #mq_properties{publisher_id = PubId,
         {ok,_Pid} ->
             case start_child_3(RefSup, Props, WorkerSeqNum, CnsProcsPerDB) of
                 ok ->
-                    start_child_2(RefSup, Props, WorkerSeqNum - 1);
-                Error ->
-                    Error
+                    start_child_2(RefSup, Props, WorkerSeqNum - 1)
```

Comment:
@mocchira It's enough to change the code without `case statement` as below, I think.

```erlang
ok = start_child_3(RefSup, Props, WorkerSeqNum, CnsProcsPerDB),
start_child_2(RefSup, Props, WorkerSeqNum - 1)
```